src/components/MultiSelectGroup/MultiSelectGroup.stories.mdx

```diff
@@ -92,3 +92,15 @@ Note: when `layout = "column"` is passed, the MultiSelect components will have `
     <stories.MultiSelectGroupLayoutStory />
   </DSProvider>
 </Canvas>
+
+## `useMultiSelect` hook
+
+`useMultiSelect` is an optional hook provided by the component, for
```

Comment:
Remove "provided by the component". The `MultiSelectGroup` does not export this hook but rather this hook stands alone. It can also be used with both `MultiSelect` and `MultiSelectGroup` so it doesn't need to be associated with only one component.

Reply:
fixed